owlify-famix/src/main/java/org/archcnl/owlify/famix/ontology/FamixOntology.java

```diff
@@ -98,6 +98,8 @@ public Individual createIndividual(OntModel model, String name) {
         private String replaceSpecialCharacters(String name) {
             // uses percent-notation, replaces spaces by '-'
             return name.replace(" ", "-")
```

Comment:
This whole method seems like a half-hearted reimplementation of https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/URLEncoder.html

The only significant difference is that spaces are replaced with `+` instead of `-`.
Newlines/carriage returns are replaced with `%0A` and `%0D`, respectively.
Does this make any difference in our usage? If not, I suggest replacing our own method completely.

Reply:
I think you're right.
In the end, we produce an URI, so the URLEncoder would be a good fit.
What do you think @nizap ?